### ISSUE TITLE
[DOC] Blocking activities has incorrect code examples

### ISSUE THREAD

**Issue by VijoPlays** [elsa-workflows/elsa-core#5338]
### Documentation Issue Overview

I've been trying to implement the Workflows according to the [documentation for Blocking activities](https://v3.elsaworkflows.io/docs/core-concepts/activity#blocking-activities).

1. From my testing it seems the workflow will only be paused if "context.CreateBookmark();" is the final statement. I added a Console.WriteLine afterwards and the entire workflow ran - after removing it, it only ran up to this activity. Is my assumption here correct, and if so, is there a way to suspend in the middle of the activity?
2. Also, the line "await workflowRunner.RunAsync(workflow, workflowState, options);" is not valid, since this particular RunAsync() overload doesn't seem to exist (anymore?). There is no RunAsync function where I could provide the workflowState and I can't find any other way of resuming the Workflow.

Would be neat if these things could be fixed - and someone mention here, which method I need to use to suspend/resume a workflow. :)

**Comment by VijoPlays** [comment]
Right, in that case the first point can be ignored (since it was mostly a question anyway).

The second point still stands and using the RunAsync method like in the example will run the entire workflow, not just resume it.

**Comment by douglasg14b** [comment]
Do you know the correct code? I'm trying to do this as well and running into trouble....

**Comment by VijoPlays** [comment]
@douglasg14b their second doc worked for me: https://elsa-workflows.github.io/elsa-documentation/custom-activities.html?section=Programmatic#blocking-activities

The IWorkflowRunTime

> workflowRuntime.TriggerWorkflowsAsync(activityTypeName, bookmarkPayload);

can execute the workflows again. 

Only done some light testing so far, but it seems that the entire Activity is getting executed after all - instead of what's been discussed so far.
